src/hooks/useSearch.js

```diff
@@ -0,0 +1,11 @@
+function useSearch(searchQuery) {
+  const fetchSearchQuery = fetch(`https://pfa.foreca.com/api/v1/location/search/${searchQuery}`, {
```

Comment:
1. It's good you're abstracting part of the logic, but would it be better make it a bit more reusable? You can pass down the specific endpoint you're targeting as a prop and that way you'll be able tp use your hook for other queries as well in the future. You could also use useState and useEffect here - this is the big advantage of the custom hooks. You can handle getting & parsing the data here as this is common for all requests. Think about what else is common for all requests and see if you can add it here.